watertap/examples/flowsheets/electrodialysis/electrodialysis_1stack_conc_recirc_ui.py

```diff
@@ -0,0 +1,634 @@
+#################################################################################
+# WaterTAP Copyright (c) 2020-2024, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National Laboratory,
+# National Renewable Energy Laboratory, and National Energy Technology
+# Laboratory (subject to receipt of any required approvals from the U.S. Dept.
+# of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#################################################################################
+from watertap.core.solvers import get_solver
+from watertap.ui.fsapi import FlowsheetInterface
+from watertap.examples.flowsheets.electrodialysis.electrodialysis_1stack_conc_recirc import (
+    build,
+    _condition_base,
+    initialize_system,
+    solve,
+)
+from pyomo.environ import units as pyunits
+
+
+def export_to_ui():
```

Comment:
You'll need to add the flowsheet to the entry points in setup.py 
![image](https://github.com/watertap-org/watertap/assets/96305519/a99eba32-6f82-46ac-b738-6da77a0677ce)